.github/workflows/slack-cli-generate-docs.yaml

```diff
@@ -0,0 +1,43 @@
+name: Generate Slack CLI Docs
+
+on:
+  workflow_dispatch:
+
+jobs:
+  update-docs:
```

Comment:
🔐 todo: This'll also need some set of permissions. Do let me know if I can help with this!